src/components/textfield.js

```diff
@@ -30,16 +30,24 @@ module.exports = function (app) {
             pattern: '',
             custom: '',
             customPrivate: false
+          },
```

Comment:
Agreed. We can always migrate if we need to. I was just trying to think of how we can structure it "future proof"